config.xml

```diff
@@ -138,6 +137,5 @@
     </plugin>
     <plugin name="cordova-plugin-inappbrowser" spec="^3.2.0" />
     <plugin name="cordova-plugin-camera" spec="^4.0.3" />
-    <engine name="ios" spec="~5.0.1" />
-    <engine name="android" spec="^8.1.0" />
+    <engine name="ios" spec="^5.0.1" />
```

Comment:
Please add the android back to supported engines. Not sure how you were able to build the app for android without this

Reply:
I didn't change these lines manually. But I can generate app-debug and also build app-release apks.
BTW Maybe it causes some problems in the future tests, so I'll add ```<engine name="android" spec="^8.1.0" />```